crates/influxive-child-svc/Cargo.toml

```diff
@@ -1,27 +1,27 @@
 [package]
 name = "influxive-child-svc"
-version = "0.0.4-alpha.1"
-edition = "2021"
 description = "Run influxd as a child process"
-license = "MIT OR Apache-2.0"
-repository = "https://github.com/holochain/influxive"
 documentation = "https://docs.rs/influxive-child-svc"
-authors = [ "Holochain Core Dev Team <devcore@holochain.org>" ]
-keywords = ["metrics"]
-categories = ["development-tools"]
+version = { workspace = true }
```

Comment:
Just a preference thing but I have tried to keep other repos with `name`, `version` and then `description` at the top in that order. Those the fields I most often look for and it's helpful to be able to find them quickly.